src/dotnet/commands/dotnet-cache/xlf/LocalizableStrings.zh-Hant.xlf

```diff
@@ -0,0 +1,108 @@
+        <target state="new">Target runtime to publish for. The default is to publish a portable application.</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="OutputOption">
+        <source>OUTPUT_DIR</source>
+        <target state="new">OUTPUT_DIR</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="OutputOptionDescription">
+        <source>Path in which to publish the app</source>
+        <target state="new">Path in which to publish the app</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="ConfigurationOption">
+        <source>CONFIGURATION</source>
+        <target state="new">CONFIGURATION</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="ConfigurationOptionDescription">
+        <source>Configuration under which to build</source>
+        <target state="new">Configuration under which to build</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="VersionSuffixOption">
+        <source>VERSION_SUFFIX</source>
+        <target state="new">VERSION_SUFFIX</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="VersionSuffixOptionDescription">
+        <source>Defines the value for the $(VersionSuffix) property in the project</source>
+        <target state="new">Defines the value for the $(VersionSuffix) property in the project</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="FxVersionOption">
+        <source>FX_Vesrion</source>
+        <target state="new">FX_Vesrion</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="FxVersionOptionDescription">
+        <source>The Microsoft.NETCore.App package version that will be used to run the assemblies</source>
+        <target state="new">The Microsoft.NETCore.App package version that will be used to run the assemblies</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="WorkingDirOption">
+        <source>WorkingDir</source>
+        <target state="new">WorkingDir</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="WorkingDirOptionDescription">
+        <source>The Directory used by the command to do its work</source>
+        <target state="new">The Directory used by the command to do its work</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="PreserveWorkingDirOption">
+        <source>Preserce WorkingDir</source>
+        <target state="new">Preserce WorkingDir</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="PreserveWorkingDirOptionDescription">
+        <source>The Directory used by the command to do its work will not be removed</source>
+        <target state="new">The Directory used by the command to do its work will not be removed</target>
+        <note></note>
+      </trans-unit>
+    </body>
+  </file>
+</xliff>
```

Comment:
It looks like my comments apply to code that was duplicated in multiple places via codegen.